<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdClient</class>
 <widget class="QMainWindow" name="AdClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="video_lb" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>611</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>611</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_lb">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>191</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>191</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weather_lb">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>191</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>210</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>210</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>广州
 25 微风   晴    </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
